build.html

```diff
@@ -77,8 +78,14 @@
       <form class="form-horizontal form-reset-password">
         <p class="instruction">Enter your new password below.</p>
         <div class="input-wrapper">
-          <input type="password" class="form-control new-password focus-outline" placeholder="Enter your new password" autocomplete="new-password" />
-          <input type="password" class="form-control confirm-password focus-outline" placeholder="Confirm your new password" autocomplete="new-password" />
+          <div class="col-sm-12 fl-new-password">
```

Comment:
Why use a Bootstrap specific `col-sm-12` class only to add `padding-left: 0` and `padding-right: 0` in CSS? It feels like we could just use a `<div>` without any Bootstrap class and avoid the padding overrides in CSS.

Though you might need to add `position: relative`.